Makefile

```diff
@@ -46,223 +21,50 @@ check-dependencies:
-	else \
-		echo "Error: just is not installed. Please install just."; \
-		exit 1; \
-	fi
 	@if command -v cargo prove >/dev/null 2>&1; then \
 		echo "cargo prove is installed."; \
 	else \
 		echo "Error: succinct is not installed. Please install SP1."; \
 		exit 1; \
 	fi
-	@if command -v operator >/dev/null 2>&1; then \
-		echo "operator is installed."; \
-	else \
-		echo "Error: operator is not installed. Please run install-dependencies."; \
-		exit 1; \
-	fi
 	@echo "All dependencies are installed."
 .PHONY: check-dependencies
 
-## demo: Run the entire demo.
-demo:
-	@make start
-	@make setup
-	@make transfer
-	@make transfer-back
-.PHONY: demo
-
 ## start: Start all Docker containers for the demo.
 start:
+	@echo "--> Starting all Docker containers"
 	@docker compose up --detach
 .PHONY: start
 
-## relay-local: Start a local instance of the hyperlane relayer (NOTE: This is temporary and should be removed in favour of Docker).
-relay-local:
-	@echo "--> Starting hyperlane relayer"
-	@cd local && CONFIG_FILES=./config/config.json ./relayer
-.PHONY: relay-local
-
-## setup: Set up the IBC light clients.
-setup:
-	@echo "--> Creating genesis.json for Tendermint light client"
-	@cd ./solidity-ibc-eureka && cargo run --quiet --bin operator --release -- genesis -o scripts/genesis.json --proof-type groth16
-	@echo "--> Creating IBC light clients"
-	@go run ./testing/demo/pkg/setup/
-.PHONY: setup
+## stop: Stop all Docker containers and remove volumes.
+stop:
+	@echo "--> Stopping all Docker containers"
+	@docker compose down -v
+.PHONY: stop
 
-## transfer: Transfer tokens from simapp to the EVM roll-up.
+## transfer: Transfer tokens from celestia-app to the EVM roll-up.
 transfer:
-	@echo "--> Transferring tokens from simapp to the EVM roll-up"
-	@go run ./testing/demo/pkg/transfer/ transfer
+	@echo "--> Transferring tokens from celestia-app to the EVM roll-up"
+	@docker run --rm \
+  		--network celestia-zkevm-hl-testnet_celestia-zkevm-net \
+  		--volume celestia-zkevm-hl-testnet_celestia-app:/home/celestia/.celestia-app \
+  		ghcr.io/celestiaorg/celestia-app-standalone:feature-zk-execution-ism \
+  		tx warp transfer 0x726f757465725f61707000000000000000000000000000010000000000000000 1234 0x000000000000000000000000d7958B336f0019081Ad2279B2B7B7c3f744Bce0a "1000" \
+  		--from default --fees 400utia --max-hyperlane-fee 100utia --node http://celestia-validator:26657 --yes
 .PHONY: transfer
 
-## transfer-back: Transfer tokens back from the EVM roll-up to simapp.
+## transfer-back: Transfer tokens back from the EVM roll-up to celestia-app.
+# TODO: https://github.com/celestiaorg/celestia-zkevm-hl-testnet/issues/34
 transfer-back:
-	@echo "--> Transferring tokens back from the EVM roll-up to simapp"
+	@echo "--> Transferring tokens back from the EVM roll-up to celestia-app"
 	@go run ./testing/demo/pkg/transfer/ transfer-back
 .PHONY: transfer-back
 
 ## query-balance: Query the balance of the receiver in the EVM roll-up.
 query-balance:
```

Comment:
Same for this, I think we can improve it later. I would expect maybe something like:
`query-balance-evm` and `query-balance-celestia` could be more useful.

Right now this uses `cast` directly to query the recv addr of the synthetic erc20 hyperlane token